lib/class-sp-api.php

```diff
@@ -123,25 +111,25 @@ public function request( $url = '', $method = 'GET', $body = '', $request_params
 			'error' => array(
 				'code' => $result->get_error_code(),
 				'message' => $result->get_error_message(),
-				'data' => $result->get_error_data()
-			)
+				'data' => $result->get_error_data(),
+			),
 		) );
 	}
 
 	public function parse_url( $url = '' ) {
-		if ( is_string( $url ) && preg_match( '#^https?://#i', $url ) )
-			return $url;
+		if ( is_string( $url ) && preg_match( '#^https?://#i', $url ) ) {
+			return $url; }
```

Comment:
It may be valid, I'm not sure, but I don't much like it regardless. `phpcbf` did these and I just didn't catch it. Thanks for the eagle eyes!